src/api/app/models/user.rb

```diff
@@ -55,6 +55,9 @@ class User < ApplicationRecord
   validates :login, :email, :password, :password_hash_type, :state,
             presence: { message: 'must be given' }
 
+  validates :password, confirmation: { message: 'password must be repeated properly.' }
```

Comment:
What is the default message? 

As long as #2868 is not done, I would stick to the default, no?